ARK Server Manager/Lib/Model/ServerProfile.cs

```diff
@@ -1889,5 +1896,25 @@ public void ResetOverrideMaxExperiencePointsDino()
         {
             OverrideMaxExperiencePointsDino = GameData.DEFAULT_MAX_EXPERIENCE_POINTS_DINO;
         }
+
+        // section reset methods
+        public void ResetChatAndNotificationSection()
+        {
+            EnableGlobalVoiceChat = GameData.DEFAULT_ENABLE_GLOBAL_VOICE_CHAT;
```

Comment:
The link you refer to is a good solution. Do you still want the constants in GameData, or scrap them and revert back to the values instead?